docs/index.md

```diff
@@ -62,72 +61,28 @@ cognee.config.data_root_directory(absolute_path_to_directory)
 #### Without .env file
 
 ```
-import os
-
-os.environ["WEAVIATE_URL"] = "YOUR_WEAVIATE_URL"
-os.environ["WEAVIATE_API_KEY"] = "YOUR_WEAVIATE_API_KEY"
 
-os.environ["OPENAI_API_KEY"] = "YOUR_OPENAI_API_KEY"
+import cognee
 
-```
+text = """Natural language processing (NLP) is an interdisciplinary
+       subfield of computer science and information retrieval"""
 
-### Run
+cognee.add(text) # Add a new piece of information
 
-Add a new piece of information to the storage:
-```
-import cognee
+cognee.cognify() # Use LLMs and cognee to create knowledge
 
-cognee.add("some_text", dataset_name)
+search_results = cognee.search("SIMILARITY", "computer science") # Query cognee for the knowledge
 
-cognee.add([
```

Comment:
Why delete this? Should we move it to some other page instead?

Reply:
probs a mistake